source/release-notes.txt

```diff
@@ -40,6 +40,7 @@ Behavioral Changes
 - Adds Debian 11 support. 
 - Adds Enterprise RHEL 8 zSeries support.
 - Removes all zSeries/IBM POWER PC platforms support except for RHEL 8 zSeries.
+- Removes RHEL 6 and RHEL 7 on S390x support.
```

Comment:
Please see this [change in Ops Manager](https://github.com/10gen/mms-docs/blob/master/source/release-notes/changelogs/ops-manager/changelog-onprem-v5.0.rst#onprem-server-507). It reads:

```
Removes support for:
RHEL 7.x/8.x, and Ubuntu 16.x on PowerPC (ppc64le) architectures.
RHEL 6.x/7.x, Ubuntu 18.x, and SUSE 12.x on zSeries (s390x) architectures.
```

Sharing because of the format of platforms that we use, and also because we might need other removals here too (if dev confirm. Dennis K from the OM team might know).